packages/dd-trace/src/proxy.js

```diff
@@ -58,6 +58,13 @@ class Tracer extends BaseTracer {
     return this
   }
 
+  setUrl (url) {
```

Comment:
Can the implementation be moved to the tracer itself? That would be more in line with how this proxy should work, would fix a possible error when calling the method before the tracer is initialized, and would avoid the private property access that may not exist on all possible implementations.

Reply:
Sure. This is done in the latest commit. With tests for everything!